src/avt/Expressions/Abstract/avtMultipleInputMathExpression.h

```diff
@@ -45,6 +45,8 @@
 
 #include <avtMultipleInputExpressionFilter.h>
 
+#include <vtkDataArray.h>
+
 class vtkDataArray;
```

Comment:
You should be able to get rid of this class declaration now that you're including the header. Everything else looks good.